packages/block-editor/src/components/off-canvas-editor/block-contents.js

```diff
@@ -77,6 +82,7 @@ const ListViewBlockContents = forwardRef(
 
 		useEffect( () => {
 			if (
+				lastInsertedBlocksActor !== 'auto' && // don't show the Link UI if the block was inserted by something other than the user.
```

Comment:
@getdave this is still not working. It prevents the first render show but when I insert a new link if I'm logging this variable I will see undefined and instantly I will see auto, because the same link I'm inserting will be synced to the other block. It still should be showing for that undefined one, I don't know why it doesn't!

Reply:
Ah that's a problem. Of course 🤦 

So when the sync happens the "last inserted blocks" becomes the ones that are synced with the entity in the "other" block. So the `actor` of the "last inserted blocks" will be `auto`.